build/teamcity/internal/release/process/make-and-publish-build-artifacts-per-platform.sh

```diff
@@ -93,7 +93,7 @@ if [[ $platform == "linux-amd64" || $platform == "linux-arm64" || $platform == "
     --ungzip \
     --ignore-zeros \
     --strip-components=1
-  cp --recursive licenses "build/deploy-${platform}"
+  cp LICENSE THIRD-PARTY-NOTICES.txt "build/deploy-${platform}"
```

Comment:
```suggestion
  cp LICENSE licenses/THIRD-PARTY-NOTICES.txt "build/deploy-${platform}"
```
I think this also should be applied to the master branch and other backports.

Reply:
Yes. It should. Hadn't gotten to it yet. You beat me to it.